packages/tracing/src/integrations/node/prisma.ts

```diff
@@ -0,0 +1,99 @@
+import { Hub } from '@sentry/hub';
+import { EventProcessor, Integration } from '@sentry/types';
+import { isThenable, logger } from '@sentry/utils';
+
+import { IS_DEBUG_BUILD } from '../../flags';
+
+type PrismaAction =
+  | 'findUnique'
+  | 'findMany'
+  | 'findFirst'
+  | 'create'
+  | 'createMany'
+  | 'update'
+  | 'updateMany'
+  | 'upsert'
+  | 'delete'
+  | 'deleteMany'
+  | 'executeRaw'
+  | 'queryRaw'
+  | 'aggregate'
+  | 'count'
+  | 'runCommandRaw';
+
+interface PrismaMiddlewareParams {
+  model?: any;
+  action: PrismaAction;
+  args: any;
+  dataPath: string[];
+  runInTransaction: boolean;
+}
+
+type PrismaMiddleware<T = any> = (
+  params: PrismaMiddlewareParams,
+  next: (params: PrismaMiddlewareParams) => Promise<T>,
+) => Promise<T>;
+
+interface PrismaClient {
+  $use: (cb: PrismaMiddleware) => void;
+}
+
+/** Tracing integration for @prisma/client package */
+export class Prisma implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'Prisma';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = Prisma.id;
+
+  /**
+   * Prisma ORM Client Instance
+   */
+  private readonly _client?: PrismaClient;
+
+  /**
+   * @inheritDoc
+   */
+  public constructor(options: { client?: PrismaClient } = {}) {
+    this._client = options.client;
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public setupOnce(_: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
+    if (!this._client) {
+      IS_DEBUG_BUILD && logger.error('PrismaIntegration is missing a Prisma Client Instance');
+      return;
+    }
+
+    this._client.$use((params: PrismaMiddlewareParams, next: (params: PrismaMiddlewareParams) => Promise<unknown>) => {
+      const scope = getCurrentHub().getScope();
+      const parentSpan = scope?.getSpan();
+
+      const action = params.action;
+      const model = params.model;
+
+      const span = parentSpan?.startChild({
+        description: `Action: ${action}, ${model ? `Model: ${model}` : ''}`,
+        op: 'prisma',
```

Comment:
```suggestion
        description: model ? `${model} ${action}` : action,
        op: 'db.prisma',
```

Is there any other additional data we can include here?

Reply:
We also have the (optional) arguments passed to the method, such as:

for `find*`
```
where: {
  email: {
    contains: 'sentry.io'
  }
}
```
or `create`
```
data: {
  name: 'Tilda',
  email: `tilda_2341@sentry.io`,
},
```

Reply:
That has PII concerns, so let’s skip this for now, we can add it back later on.